backend/app/api/v1/endpoints/custom_sites.py

```
@@ -0,0 +1,312 @@
+        db.refresh(db_site)
+        logger.info(f"Successfully created custom site: {db_site.id}")
+        return db_site
+        
+    except HTTPException:
+        raise
+    except Exception as e:
+        db.rollback()
+        logger.error(f"Error creating custom site: {str(e)}")
+        logger.error(f"Error type: {type(e).__name__}")
+        logger.error(f"Site data: {site_data}")
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)